src/DonateHope.WebAPI/Controllers/v1/CampaignContribution/CampaignContributionController.cs

```diff
@@ -97,6 +97,23 @@ public async Task<ActionResult<CampaignContributionGetResponseDto>> GetCampaignC
         }
         return result.Value;
     }
+    
+    [HttpGet("campaign-contributions/{campaignId}", Name = nameof(GetCampaignContributionsByCampaignId))]
```

Comment:
1. I see in the current code, we already had:
[HttpGet("{id}", Name = nameof(GetCampaignContribution))]
    public async Task<ActionResult<CampaignContributionGetResponseDto>> GetCampaignContribution([FromRoute] Guid id)
   
   under controller: 
[ApiVersion("1.0")]
[Route("api/v{version:apiVersion}/campaign-contribution")]
[ApiController]
public class CampaignContributionController(...

=> what is the difference?

2. We should stick to the current convention using path singularly:
campaign-contributionS --> campaign-contribution

Reply:
thanks for your comment.
1. The current one: `public async Task<ActionResult<CampaignContributionGetResponseDto>> GetCampaignContribution([FromRoute] Guid id)` --> this is to get information of one CampaignContributon
The new one: 
```
[HttpGet("campaign-contributions/{campaignId}", Name = nameof(GetCampaignContributionsByCampaignId))]
    public async Task<ActionResult<IEnumerable<CampaignContributionGetResponseDto>>> GetCampaignContributionsByCampaignId([FromRoute] Guid campaignId)
```
this API is to get list of contributions belong to a campaign.
That's why I am using plural nouns for the endpoint